common/cert.c

```diff
@@ -211,6 +211,36 @@ oe_result_t oe_get_crl_distribution_points(
     return result;
 }
 
+oe_result_t oe_cert_chain_get_root_cert(
+    const oe_cert_chain_t* chain,
```

Comment:
Can these functions be called from the enclave? Can a plugin just use headers under crypto. Just wondering why there was a change to include the internal cert.h rather than crypto cert.h

Reply:
Yes, they can and they are defined in internal/crypto/cert.h. The change to the headers is unnecessary and will be reverted.